virttest/test_setup.py

```diff
@@ -1284,20 +1286,20 @@ def get_same_group_devs(self, pci_id):
             pci_ids = out.split()
         return pci_ids
 
-    def get_pf_mlx(self):
+    def get_pf_ifname(self, pci_pf):
```

Comment:
we can remove this function, we can use `get_interface_from_pci_id()` from utils_misc.

Reply:
Thanks, updated, please help review again.